charts/openyurt/templates/pool-coordinator.yaml

```diff
@@ -0,0 +1,209 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: pool-coordinator-apiserver
+  namespace: {{ .Release.Namespace | quote }}
+  annotations:
+    openyurt.io/topologyKeys: openyurt.io/nodepool
+  labels:
+    name: pool-coordinator
+spec:
+  type: ClusterIP
+  ports:
+    - port: 443
+      targetPort: {{ .Values.poolCoordinator.apiserverPort }}
+      protocol: TCP
+      name: https
+  selector:
+    k8s-app: pool-coordinator
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: pool-coordinator-etcd
+  namespace: {{ .Release.Namespace | quote }}
+  annotations:
+    openyurt.io/topologyKeys: openyurt.io/nodepool
+  labels:
+    name: pool-coordinator
+spec:
+  type: ClusterIP
+  ports:
+    - port: 2379
+      targetPort: {{ .Values.poolCoordinator.etcdPort }}
+      protocol: TCP
+      name: https
+  selector:
+    k8s-app: pool-coordinator
+---
+apiVersion: apps.openyurt.io/v1alpha1
+kind: YurtAppDaemon
+metadata:
+  name: pool-coordinator
+  namespace: {{ .Release.Namespace | quote }}
+spec:
+  selector:
+    matchLabels:
+      k8s-app: pool-coordinator
+  workloadTemplate:
+    deploymentTemplate:
+      metadata:
+        labels:
+          k8s-app: pool-coordinator
+      spec:
+        replicas: 1
+        selector:
+          matchLabels:
+            k8s-app: pool-coordinator
+        template:
+          metadata:
+            labels:
+              k8s-app: pool-coordinator
+          spec:
+            containers:
+              - command:
+                  - kube-apiserver
+                  - --bind-address=0.0.0.0
+                  - --allow-privileged=true
+                  - --anonymous-auth=true
+                  - --authorization-mode=Node,RBAC
+                  - --client-ca-file=/etc/kubernetes/pki/ca.crt
+                  - --enable-admission-plugins=NodeRestriction
+                  - --enable-bootstrap-token-auth=true
+                  - --disable-admission-plugins=ServiceAccount
+                  - --etcd-cafile=/etc/kubernetes/pki/etcd/ca.crt
+                  - --etcd-certfile=/etc/kubernetes/pki/apiserver-etcd-client.crt
+                  - --etcd-keyfile=/etc/kubernetes/pki/apiserver-etcd-client.key
+                  - --etcd-servers=http://127.0.0.1:{{ .Values.poolCoordinator.etcdPort }}
+                  - --kubelet-client-certificate=/etc/kubernetes/pki/apiserver-kubelet-client.crt
+                  - --kubelet-client-key=/etc/kubernetes/pki/apiserver-kubelet-client.key
+                  - --kubelet-preferred-address-types=InternalIP,ExternalIP,Hostname
+                  - --secure-port={{ .Values.poolCoordinator.apiserverSecurePort }}
+                  - --service-account-issuer=https://kubernetes.default.svc.cluster.local
+                  - --service-account-key-file=/etc/kubernetes/pki/sa.pub
+                  - --service-account-signing-key-file=/etc/kubernetes/pki/sa.key
+                  - --service-cluster-ip-range=10.96.0.0/12
```

Comment:
parameter `--service-cluster-ip-range` should be match witch kube-apiserver on cloud, so end user should configure this setting in `values.yaml`